pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -78,41 +93,63 @@ export default Vue.extend<Data, Methods, Computed, never>({
         this.$store.dispatch('snapshots/fetch');
       }, 1500);
     },
+    setCardsHeight() {
+      /**
+       * height = window height - page header - space reserved for snapshot event
+       */
+      this.cardsStyle = `height: ${ window?.innerHeight - 150 - (this.snapshotEvent ? 75 : 0) }px`;
+    },
+    toggleMainScrollbar(value: boolean) {
+      const main = document.getElementsByTagName('main')?.[0];
+
+      if (main) {
+        main.style.overflowY = value ? 'auto' : 'hidden';
+      }
+    },
```

Comment:
I have a couple of concerns with this approach of toggling the scrollbar. Overall, I think that it might be beneficial to reconsider manipulating styles directly in javascript from `Snapshots.vue`:

1. Explicitly signaling when a child component is modifying parent styles is crucial for maintainability
2. Toggling the scrollbar in javascript might hurt readability and long-term maintenance

If we choose to pursue this route, perhaps it could be done closer to the template where 'main' is defined. Ultimately, I think that a more css-centric solution should be applied to achieve the desired behavior in this PR.

Reply:
Page sizing handling via Javascript is totally removed.